molgenis-questionnaires/src/main/frontend/src/store/index.js

```diff
@@ -0,0 +1,65 @@
+import Vue from 'vue'
```

Comment:
i would suggest not using index.js in molgenis , it makes is hard to find files as there are a lot of index.js files. This idea ( i do not like it in general) only really works for smaller ( modules) project imo.